src/Microsoft.AspNet.Razor.Runtime/TagHelpers/TagHelperAttribute.cs

```diff
@@ -42,6 +42,13 @@ public TagHelperAttribute(string name, object value)
         public object Value { get; set; }
 
         /// <summary>
+        /// Gets or sets the value indicating if the attribute is minimized.
```

Comment:
`Gets or sets an indication whether the attribute is minimized or not.` i.e. consistent with `IReadOnlyTagHelperAttribute` property.